forte.yml

```diff
@@ -301,8 +301,9 @@ Layer:
     <<: *db
     table: |-
         ( SELECT
-            geometry, '{lang}' as lang, type, capital, NULL as ldir,
+            geometry, '{lang}' as lang, type, NULL as ldir,
             CASE WHEN type='city' THEN 1 WHEN type='state' THEN 2 ELSE 100 END AS prio,
+            CASE WHEN capital='yes' THEN 1 WHEN capital IN ('4','5','6','7','8','9','10') THEN capital::INT ELSE 100 END AS capital,
```

Comment:
There are a bunch of `capital=2` around, it seems, for islands. I think we should deal with that.

Reply:
You are right, I looked only in my database (with all Europe) and there are 9 with `capital=2` and 37 with `capital=3`

I should use more the community tools, cf https://taginfo.openstreetmap.org/keys/capital#values

Should be ok with: https://github.com/tilery/pianoforte/pull/45/commits/e7f34b0a8612013095545f9a973b4fccbae121af